### PR TITLE
added changes for the update package kind-of: 6.0.3

### PR DESCRIPTION
**Pull request by sumanprogress**
Signed-off-by: sumanprogress jiddiSuman.Yadav@progress.com

### :nut_and_bolt: Description: What code changed, and why?
We need to Upgrade kind-of: 6.0.3 or later

### :chains: Related Resources
#5607

### :+1: Definition of Done
I have updated the Kind-of package version.

### :white_check_mark: Checklist

**All PRs** must tick these:

- [x] I have read the [CONTRIBUTING document](https://github.com/chef/automate/blob/master/CONTRIBUTING.md).
- [x] All commits signed-off for the [Developer Certification of Origin](https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco).

With occasional exceptions, all PRs **from Progress employees** must tick these:

- [x] Is the code clear? *(complicated code or lots of comments--subdivide and use well-named methods, meaningful variable names, etc.)*
- [x] Consistency checked? *(user notifications, user prompts, visual patterns, code patterns, variable names)*
- [x] Repeated code blocks eliminated? *(adapt and reuse existing components, blocks, functions, etc.)*
- [x] Spelling, grammar, typos checked? *(at a minimum use `make spell` in any component directory)*
- [x] Code well-formatted? *(indents, line breaks, etc. improve rather than hinder readability)*

All PRs **from Progress employees** should tick these if appropriate:

- [x] Tests added/updated? (all new code needs new tests)
- [x] Docs added/updated? (all customer-facing changes)

*Please add a note next to any checkbox above if you are NOT ticking it.*

### CODE REVIEW

**Comment by chaitali-mane** [comment]
@sumanprogress update 'kind-of' package version in package-lock.json, not in package.json

File: components/chef-ui-library/package-lock.json [hunk 1]
```diff
@@ -8107,4 +8107,4 @@
       }
     }
   }
-}
+}
```
Comment (vinay033) [inline components/chef-ui-library/package-lock.json:8110]: add end of line in this file

**Comment by vinay033** [comment]
@sumanprogress  have you done these changes manually or used the command if you did it manually then do using the command.

**Review by vinay033 — CHANGES_REQUESTED** [review]

File: components/chef-ui-library/package-lock.json (hunk 1 above)
Comment (sumanprogress) [reply components/chef-ui-library/package-lock.json]: new line added

**Review by chaitali-mane — APPROVED** [review]

**Review by vinay033 — APPROVED** [review]

**Pull request merged**